functionsandconditionals.js

```diff
@@ -0,0 +1,66 @@
+//Biggest Numbers
+//1
+function maxOfTwoNumbers(num1 , num2){
+   if( num1 > num2){
+        console.log('num1 is larger than num2');
+    } else{
+       console.log('num2 is larger than num1');
+    }
+}
+maxOfTwoNumbers( 578 , 587 );
+
+//2
+function maxOfThree( num1 , num2 , num3 ){
+   console.log(Math.max(num1 , num2 , num3));
+}
+maxOfThree ( 145 , 164 , 136);
+
+
+//The world translator
+
+var language1 = 'English';
+var language2 = 'Arabic';
+var language3 = 'French';
+
+function helloWorld(language){
+    if(language.language1 === 'English'){
```

Comment:
` if(language === 'English')` the language here is a parameter and the value of this parameter is the one you pass it when you called the function `helloWorld('English');` don't forget to comment  using `//` or add `"` or `'` between any letter or word that it is not a keyword in javascript or not declared as a variable or a function so your machine doesn't get confused and throw undefined errors